frontend/components/site/siteBuilds.js

```diff
@@ -91,7 +91,7 @@ class SiteBuilds extends React.Component {
 
               switch (build.state) {
                 case 'error':
-                  message = build.error;
```

Comment:
Question: I thought the entire table was being loaded at once? This suggests that we can check the length of the error and only return if its under X size......

Reply:
And the change in api/serializers/build.js fixes the crash?